### ISSUE TITLE
Expose API for checking if job is complete

### ISSUE THREAD

**Issue by jszwedko** [modcloth/queued-command-runner#8]
It feels like it would be useful to be able to determine of the job we queued finished rather than if the queue was completely processed. Maybe consider returning a channel from `Run` that can be used.

**Comment by rafecolton** [comment]
Yep, returning a channel from `Run` seems like the best way to go.  Will implement.
